CMakeLists.txt

```diff
@@ -380,14 +385,15 @@ macro(nebula_link_libraries target)
         boost_system
         event
         double-conversion
-        resolv
         s2
         ${OPENSSL_SSL_LIBRARY}
         ${OPENSSL_CRYPTO_LIBRARY}
         ${KRB5_LIBRARIES}
         ${COMPRESSION_LIBRARIES}
         ${JEMALLOC_LIB}
         ${LIBUNWIND_LIBRARIES}
+        keyutils
```

Comment:
I try this pr locally,   it seems we don't need keyutils currently.  
Does it being introduced by some other libraries?

Reply:
It's not needed if it's not present while compiling third party. Otherwise, krb5 will link against it if found.

Reply:
This is the very part annoying me much.